src/components/TopMenu.js

```diff
@@ -1,25 +1,31 @@
-import React from 'react'
+import * as React from 'react'
 import { Link, graphql } from 'gatsby'
 import { RichText } from 'prismic-reactjs'
-import LanguageSwitcher from './LanguageSwitcher'
-import logo from '../images/logo.png'
+import { StaticImage } from 'gatsby-plugin-image'
 
-const TopMenu = ({ topMenu, activeDocMeta }) => {
+import { LanguageSwitcher } from './LanguageSwitcher'
+
+export const TopMenu = ({ topMenu, activeDocMeta }) => {
   const renderedMenuLinks = topMenu.menu_links
     ? topMenu.menu_links.map((menuLink, index) => (
-      <li key={`top-nav-${index}`}>
-        <Link id={menuLink.link.id} to={menuLink.link.url}>
-          {RichText.asText(menuLink.label.raw)}
-        </Link>
-      </li>
-    ))
+        <li key={`top-nav-${index}`}>
+          <Link id={menuLink.link.id} to={menuLink.link.url}>
```

Comment:
Is there a reason why there is a space/tag in the beggening?

Reply:
The code has been formatted by Prettier automatically so we don't have to worry about code style. In this case, Prettier formats long ternaries like this:

```js
condition
  ? trueExpression
  : falseExpression
```

For this specific case, the JSX tags are indented an extra level since it is wrapped within a `map` callback:

```js
array.map((element) => (
  <JSXExpression />
)
```

The combination of both conditions results in the style of indention. But because it's formatted with Prettier, it will automatically format should the code change in the future.